app/controllers/theme_javascripts_controller.rb

```diff
@@ -34,6 +34,31 @@ def show
     send_file(cache_file, disposition: :inline)
   end
 
+  def show_tests
+    raise Discourse::NotFound if Rails.env.production?
```

Comment:
Thank you David! I only added this check because the `/qunit` route does the same check, and yeah it's definitely temporarily and I'll revisit when theme testing is a little more fleshed out (didn't want to do everything at once).

My plan is to add a hidden site setting that controls who can run JS tests. Allowed values for that setting would be `staff` and `all`. Theme creator would then be on `all` and we would need to do some patching to allow the user to only run tests of themes they own.

Rake task to run a theme's tests will definitely be added as well 👍 ❤️